.rspec

```diff
@@ -1,4 +1,4 @@
 --color
 --backtrace
 --require spec_helper.rb
---pattern="spec/*_spec.rb","spec/blast_versions/blast_2.9.0/*.rb"
```

Comment:
That can be in a separate PR, maybe?
It is needed for running test on specific files. That was not enabled before and very helpful for writing tests since otherwise the whole test-suite was always running during local testing with for example: `bundle exec rspec spec/job_spec.rb`